PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -86,21 +98,29 @@ ${CARET7DIR}/wb_command -volume-palette $Jacobian MODE_AUTO_SCALE -interpolate t
 # To detect this situation, and prevent it from occurring, FreeSurferPipeline errors out if the
 # one-step warpfield created here ($OutputOrigT2wToT1w) already exists.]
 convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT1wTransform" --warp1="$dcT1wTransform" --out="$OutputOrigT1wToT1w"
-convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT2wTransform" --warp1="$dcT2wTransform" --postmat="$FinalT2wTransform" --out="$OutputOrigT2wToT1w"
+
+if [ "${T2wPresent}" = "YES" ] ; then
+  convertwarp --relout --rel --ref="$T1wImageBrainMask" --premat="$InitialT2wTransform" --warp1="$dcT2wTransform" --postmat="$FinalT2wTransform" --out="$OutputOrigT2wToT1w"
+fi
 
 applywarp --rel --interp=spline -i "$OrginalT1wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT1wToT1w" -o "$OutputT1wImage"
 fslmaths "$OutputT1wImage" -abs "$OutputT1wImage" -odt float
 fslmaths "$OutputT1wImage" -div "$BiasField" "$OutputT1wImageRestore"
 fslmaths "$OutputT1wImageRestore" -mas "$T1wImageBrainMask" "$OutputT1wImageRestoreBrain"
 
-applywarp --rel --interp=spline -i "$OrginalT2wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT2wToT1w" -o "$OutputT2wImage"
-fslmaths "$OutputT2wImage" -abs "$OutputT2wImage" -odt float
-fslmaths "$OutputT2wImage" -div "$BiasField" "$OutputT2wImageRestore"
-fslmaths "$OutputT2wImageRestore" -mas "$T1wImageBrainMask" "$OutputT2wImageRestoreBrain"
+if [ "${T2wPresent}" = "YES" ] ; then
+  applywarp --rel --interp=spline -i "$OrginalT2wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT2wToT1w" -o "$OutputT2wImage"
+  fslmaths "$OutputT2wImage" -abs "$OutputT2wImage" -odt float
+  fslmaths "$OutputT2wImage" -div "$BiasField" "$OutputT2wImageRestore"
+  fslmaths "$OutputT2wImageRestore" -mas "$T1wImageBrainMask" "$OutputT2wImageRestoreBrain"
+fi
 
 # Do the same for the equivalents in MNINonLinear space
 convertwarp --relout --rel --ref="$T1wImageBrainMask" --warp1="$OutputOrigT1wToT1w" --warp2="$AtlasTransform" --out="$OutputOrigT1wToStandard"
-convertwarp --relout --rel --ref="$T1wImageBrainMask" --warp1="$OutputOrigT2wToT1w" --warp2="$AtlasTransform" --out="$OutputOrigT2wToStandard"
+
+if [ "${T2wPresent}" = "YES" ] ; then
+  convertwarp --relout --rel --ref="$T1wImageBrainMask" --warp1="$OutputOrigT2wToT1w" --warp2="$AtlasTransform" --out="$OutputOrigT2wToStandard"
+fi
```

Comment:
Ditto.  Consolidate with the `applywarp` command for the MNI space.

Reply:
Done.